geniesp/bpc_redcap_export_mapping.py

```diff
@@ -509,7 +509,7 @@ class BpcProjectRunner(metaclass=ABCMeta):
     # Redcap codes to cbioportal mapping synid and form key is in
     # version 38 was the last stable version
     # Use version 42 - but there is a bug in Synapse...
-    _REDCAP_TO_CBIOMAPPING_SYNID = "syn25712693.42"
+    _REDCAP_TO_CBIOMAPPING_SYNID = "syn25712693"
```

Comment:
Prior to merge, the mapping file needs to be updated to use a particular version so all mapping issues need to be resolved in proper channels.